x-pack/plugins/monitoring/server/lib/metrics/logstash/classes.js

```diff
@@ -395,6 +416,14 @@ export class LogstashPipelineNodeCountMetric extends LogstashMetric {
         );
       });
 
+      const oldNodesCount = _.get(bucket, 'no_pipelines.node_count.value', 0);
```

Comment:
I wrote a comment like this to explain what's happening below, but I'm not sure if it's necessary. Would like some input.

```
/**
  * In older versions of Logstash there is no pipelines field for the
  * nested aggs above to tally, so we add 1 to the count for each
  * Logstash instance writing to the index without that field.
*/
```

Reply:
Personally, future me would very much appreciate a comment like this explaining the "why" of the code here. I'd suggest adding to your comment something about why there is no `pipelines` field in older versions of Logstash: because back then 1 instance == 1 pipeline.